opensfm/commands/export_visualsfm.py

```diff
@@ -57,6 +57,14 @@ def image_path(self, image, data):
         return os.path.relpath(path, data.data_path)
 
     def image_size(self, image, data):
-        """Height and width of the undistorted image."""
-        image = data.load_undistorted_image(image)
-        return image.shape[:2]
+        """
+        Height and width of the undistorted image.
+        """
+        try:
+            file_path = data._undistorted_image_file(image)
+            with Image.open(file_path) as img:
+                return img.size
```

Comment:
```suggestion
                width, height = img.size
                return height, width
```

PIL seems to return `width, height` while we are returning `height, width`